src/environment.cpp

```diff
@@ -1064,7 +1064,8 @@ void ServerEnvironment::step(float dtime)
 	/*
 		Manage active block list
 	*/
-	if(m_active_blocks_management_interval.step(dtime, 2.0))
+	float abm_interval = g_settings->getFloat("abm_interval");
```

Comment:
That would make it unchangeable at runtime.

I asked specifically on IRC and got this reply from @celeron55:

> sofar: i would say calling it exactly once per setting for every step is fine, and even recommended so that the settings can be updated at runtime

Notice I'm properly reusing the setting in the rest of the function, so, I think this is appropriate. (I referenced your "cache a few settings" patch for this, as well, so I'm aware that eliminating settings-gets is also desirable).